nx/source/services/acc.c

```diff
@@ -108,6 +109,38 @@ Result accountListAllUsers(u128* userIDs)
     return rc;
 }
 
+Result accountListAllUsers(u128* userIDs, size_t max_userIDs, size_t *actual_total)
+{
+    Result rc=0;
+
+    u128 *temp_userIDs;
+	temp_userIDs = malloc(sizeof(u128) * ACC_USER_LIST_SIZE);
```

Comment:
Tab.

Reply:
Oops... that was silly of me, I think it should be fixed now.